download/download-pages.rkt

```diff
@@ -58,6 +58,11 @@
                        " text-align: left;"
                        " line-height: 1.5em; "
                        " background-color: #edd"))
+  (define (upgrade-advice-packages)
+    @p{@strong{Upgrading? } When you freshly install a new version, no package from
+       your previous version will be available. You can either use @tt{raco pkg 
+       migrate} (e.g. @tt{raco pkg migrate 8.10}) or “Copy from Version” in DrRacket 
```

Comment:
It might be worth changing `raco pkg migrate` to `raco pkg migrate <previous-version>`. In the current form, readers might not know what 8.10 is.